showcase/src/app/app.component.html

```diff
@@ -54,8 +54,7 @@
 					</button>
 				</div>
 			</div>
-
-			<div class="stark-app-bar-page-title">
+			<div class="stark-header-extra">
```

Comment:
`stark-header-extra` class does not exist.
Could you please revert this part of the code as it was before ?

Reply:
Also, could you please integrate `stark-app-logout` component at line 33 in replacement of 
```html
<button color="alert" mat-mini-fab>
    <mat-icon starkSvgViewBox svgIcon="power" [matTooltip]="'STARK.APP_LOGOUT.TITLE' | translate"></mat-icon>
</button>
``` 
which is a placeholder in waiting of the final stark-app-logout component that you developed.